Updated information on storage.

jamesche75 opened the pull request:
Changed WEBSITES_ENABLE_APP_SERVICE_STORAGE doc so that it is obvious it is "false" by default.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jamesche75 committed:
Updated information on storage.

Changed WEBSITES_ENABLE_APP_SERVICE_STORAGE doc so that it is obvious it is "false" by default.

ahmedelnably reviewed (approved):
Thanks for your valuable contribution
